script.js

```diff
@@ -1,4 +1,213 @@
+// Steps
+// 1) Generate Random Number of two dices max 6, return output for player 1
+// 2) Player choose which number to go first, return order output for player 1
+// 3) Generate Random Number of two dices max 6, return output for player 2
+// 4) Player choose which number to go first, return order output for player 2
+// 5) Compare two order outputs, return output which player wins
+// 6) Reset the Game without refreshing the browser page
+
+var gameMode = "startGame";
+var myOutputValue = "";
+var getDiceNumber1;
+var getDiceNumber2;
+var getDiceNumber3;
+var getDiceNumber4;
+var playerNumberCombination1;
+var playerNumberCombination2;
+
+//generate random dicenumber
+var generateRandomDiceNumber = function () {
+  var randomDecimal = Math.random() * 6;
+  var diceNumber = Math.ceil(randomDecimal);
+  return diceNumber;
+};
+
+//player 1 get two dice numbers
+var getTwoDiceRollResult1 = function () {
+  getDiceNumber1 = generateRandomDiceNumber();
+  getDiceNumber2 = generateRandomDiceNumber();
+  var twoDiceRollResult = `The dice rolls are ${getDiceNumber1} and ${getDiceNumber2} respectively. <br><br> 
+  You now pick the order of the dice you want so as to make a higher combination to win. <br>
+  Insert <b>'1'</b> to go with the first number first and <b>'2'</b> to go the the second number first.`;
+  return twoDiceRollResult;
+};
+
+//player 1 pick which number goes first
+var pickNumberToGoFirst1 = function (input) {
```

Comment:
I think there's quite a bit of repeated code as well for `pickNumberToGoFirst1` and `pickNumberToGoFirst2`, try to create just one fucntion that is reusable for both player1 and 2